templates/apollo-runtime-ts/README.md

```diff
@@ -0,0 +1,44 @@
+## apollo-starter-ts
+
+Template that showcases Graphback runtime generation capabilities using 
+resolver layer generated in memory. 
+
+### Example 
+
+To use runtime capabilities developers will need to create data provider.
+Currently 2 data providers are supported:
+
+ - KnexDBDataProvider
+ - PgKnexDBDataProvider
+ - RestDataProvider (Looking for contributions)
+
+Then developers can create runtime instance:
+
+```ts
+    const backend = new GraphQLBackendCreator(`your model string here`, {});
+    const dbClientProvider = new PgKnexDBDataProvider(client);
+    const runtime = await backend.createRuntime(dbClientProvider, pubSub);
+    console.log(runtime.schema,  runtime.resolvers)
+```    
+
+See: `./runtime.ts` for fully functional example.
```

Comment:
Maybe this should link to the runtime file?

Suggestion:

See [`./runtime.ts`](https://github.com/aerogear/graphback/blob/84200cd56b9f531db30ea4fb3d8d86555a449b00/templates/apollo-runtime-ts/src/runtime.ts#L30) for a fully functional example.